pipdeptree.py

```diff
@@ -595,10 +602,15 @@ def main():
             return_code = 1
 
     show_only = set(args.packages.split(',')) if args.packages else None
+    exclude = set(args.exclude.split(',')) if args.exclude else []
+
+    if show_only and exclude:
```

Comment:
`show_only` and `exclude` can be used together as long as there are no packages in common between them. One use case of this is for eg. to show all dependencies of only "Flask" but except "Jinja2". I think it should be sufficient to validate that `show_only` and `exclude` are disjoint sets. It would be good to have a test case for this too.